Makefile

```diff
@@ -64,12 +64,8 @@ endif
 
 all: manager
 
-# Run tests & validate against linters
-.PHONY: check
-check: test lint-golint
-
 # Run tests
-test: generate fmt vet manifests
+test: generate fmt vet lint-golint manifests
```

Comment:
FYI, this change is going to run golint-ci twice now, once in `make test` as well as once for the GH action (https://github.com/projectcontour/contour-operator/blob/main/.github/workflows/prbuild.yaml#L25-L33).

Reply:
@stevesloka thanks for catching this. Commit `7db0230` keeps the existing make targets and simply updates the contrib guide to use the appropriate target.

Reply:
If you prefer to use `make test` we can swap things around. I'll merge this and can follow up with another PR if desired.